wandb/sdk/wandb_settings.py

```diff
@@ -87,7 +87,11 @@ def _str_as_bool(val: Union[str, bool, None]) -> bool:
         return ret_val
     except (AttributeError, ValueError):
         pass
-    raise UsageError(f"Could not parse value {val} as a bool")
+    except Exception:
```

Comment:
is this right, or should the warn just be in the fallthrough case?

Reply:
yea, fixed.